internal/schema-registry/schema_utilities.go

```diff
@@ -64,7 +64,7 @@ func RegisterSchemaWithAuth(cmd *cobra.Command, schemaCfg *RegisterSchemaConfigs
 			return 0, err
 		}
 	} else {
-		output.Printf(errors.RegisteredSchemaMsg, response.Id)
+		output.Printf(false, errors.RegisteredSchemaMsg, response.Id)
```

Comment:
I'm a bit unclear still on when we use `c.Config.EnableColor` and when we hardcode `false`.

Reply:
The majority of the time, `c` is not available and making it available would result in a huge rewrite. My plan is to gradually rewrite these one by one. For the time being, we use `false` which corresponds to no highlighting, even when colors are enabled.

Reply:
Oh woops, you addressed this last time. Gotcha. Lgtm after the other comment.